src/github/client/github-installation-client.ts

```diff
@@ -207,10 +214,17 @@ export class GitHubInstallationClient {
 
 	public async getNumberOfReposForInstallation(): Promise<number> {
 		const response = await this.graphql<{ viewer: { repositories: { totalCount: number } } }>(ViewerRepositoryCountQuery);
-
 		return response?.data?.data?.viewer?.repositories?.totalCount;
 	}
 
+	// TODO - remove this when we have replaced Probot and can make graphql requests for both cloud and server
+	public async getNumberOfReposForInstallationRest(): Promise<number> {
```

Comment:
Graphql (imported by Probot) fail for GHE at this point. We can clean this up once we have our own webhook receiver and have configured graphql to work with cloud and server.

Reply:
imported by probot?  This is our own implementation though... this should work with GHE.  If it doesn't, we have a bigger issue that the APIs are not consistent between github.com and ghe

Reply:
configuring what?  Graphql?  look at the graphql function in the class